di_website/publications/models.py

```diff
@@ -244,6 +272,7 @@ def get_context(self, request):
         ).distinct().order_by('name')
         context['selected_topic'] = topic_filter
         context['countries'] = Country.objects.all().order_by('region', 'name')
+        context['resource_types'] = PublicationType.objects.all().order_by('resource_category', 'name')
```

Comment:
Shouldn't you also be keeping track of the selected resource type somewhere here? Also, perhaps group them better ... why u implanting this between countries and selected_country? hehe

Reply:
Sure thing, added selected_resource_type variable